### PR TITLE
Final MileStone

### PR DESCRIPTION
**Pull request by digia3d**
This is a React capstone project. For this project I have done:

Day 1

    Selected the Weather App  API.
    Built file structure for my React app.
    Prepared routes and navigation for my app.

Day 2

    display a list of items and filter them. Data come from the API.
    Created the tests for the application.

Day 3

    Styled my components to match the design provided.
    Deployed the project and tested for final details.
    Recorded a video for my project.
    Created a good README

[Loom represantation](https://www.loom.com/share/fe125f2fdf214cfd8bccbae31ca26eac)

### CODE REVIEW

File: src/components/redux/Weathers.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    pressure: obj.main.pressure,
+    humidity: obj.main.humidity,
+    speed: obj.wind.speed,
+    deg: obj.wind.deg,
+    gust: obj.wind.gust,
+    all: obj.clouds.all,
+    country: obj.sys.country,
+    sunrise: obj.sys.sunrise,
+    sunset: obj.sys.sunset,
+    timezone: obj.timezone,
```
Comment (Wes-Isaac) [inline src/components/redux/Weathers.js]: - [x] Great job so far, kindly remove all commented code to make your code more readable.

File: src/reportWebVitals.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
```
Comment (Wes-Isaac) [inline src/reportWebVitals.js]: - [x] Kindly remove all unused files that are generated when creating the app with `create-react-app`.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @digia3d, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Deepakdanger — APPROVED** [review]
Hi @digia3d, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by digia3d** [comment]
I MADE IT!

**Pull request merged**